Formula/getmail6.rb

```diff
@@ -0,0 +1,24 @@
+class Getmail6 < Formula
+  include Language::Python::Shebang
+
+  desc "Extensible mail retrieval system with POP3, IMAP4, SSL support"
+  homepage "https://github.com/getmail6/getmail6"
+  url "https://github.com/getmail6/getmail6/archive/refs/tags/v6.18.12.tar.gz"
+  sha256 "419dc11184b147ef4daac2ec9f136b1b37e6450e2f44099629a6b7650deb044a"
+  license "GPL-2.0-only"
+
+  uses_from_macos "python", since: :catalina
+
+  def install
+    files = %w[getmail getmail_fetch getmail_maildir getmail_mbox]
+    rewrite_shebang detected_python_shebang(use_python_from_path: true), *files
+    libexec.install files
+    bin.install_symlink Dir["#{libexec}/*"]
+    libexec.install "getmailcore"
+    man1.install Dir["docs/*.1"]
```

Comment:
```suggestion
    man1.install Pathname.glob("docs/*.1")
```
These aren't directories, so let's avoid `Dir.[]`.